sandbox/extract-imagery-metadata_mg.R

```diff
@@ -84,58 +86,28 @@ image_count
 
 
 #START CODE FOR CENTROID LAT/LONG
-
-GPSLocation = exif$GPSPosition
-
-#splitting the list (was this step really necessary for my dataset? I'm not really sure this actually helped?)
-coordinates <- strsplit(GPSLocation, split = ";")
-
-# Separating the GPS from the whole exif file
-almostdone <- strsplit(unlist(coordinates), split = ",")
-
-#Creating a seperate dataframe with just the coordinates to call upon later
-isolatedcoordinates <- as.data.frame(do.call(rbind, almostdone))
-
 library(stringr)
-#Need to download this package as it will help with the following functions
-separator(df)
-
-install.packages("tidyverse")
 library(tidyverse)
 
-df2 = separate(exif,
-        col = GPSPosition,
-        into = c("lat", "long"),
-        sep = " " )
-#Here this worked, instead of doing by number of characters, did it by seperating at
-#the space in between the two (since the number of characters differed from row to row).
+#Getting the coordinates listed in the exif file and for it to seperate into long/lat
 
-print(df2)
-#printed out the new df that includes the "lat" and "long"
+coordinate = sf::st_coordinates(exif)
+lon = coords[, 1]
+lat = coords[, 2]
```

Comment:
You no longer need `lon` or `lat`, and also the `coords` object is not defined (only `coordinate` is defined) so these lines generate errors. You can delete these lines.

Reply:
This change was not made